packages/tables/src/Actions/Action.php

```diff
@@ -35,7 +36,11 @@ public function getAlpineClickHandler(): ?string
             return null;
         }
 
-        return $this->generateJavaScriptClickHandler('mountAction');
```

Comment:
My eye fell upon this in the `getAlpineClickHandler()`, this is inside a table action. Is it correct that this should be `mountAction()` indeed, or should this be `mountTableAction()` too?

Reply:
I think this should be `mountTableAction`

Reply:
But potentially it needs to be `$wire.`? What is is like in the main Action class?

Reply:
In the `Actions\StaticAction` it is `close()`, and for notification actions it is `markAsRead()` or `markAsUnread()`.